Comfy/app.css

```diff
@@ -506,12 +506,6 @@
 :root .Root__now-playing-bar .playback-bar .saber-hilt {
   height: 0;
 }
-:root .Root__now-playing-bar .playback-bar .playback-bar__progress-time-elapsed {
```

Comment:
Not sure what this is doing here to be entirely honest, it was in my commit history from a month ago so i'm assuming it's important

Reply:
Just revert this since it has nothing to do with this PR and we're good to go

Reply:
@kyrie25 I'm not sure that's a good idea, it seems this is also the most recent commit to the main repo, reverting it might revert that as well. Ultimately this change is meaningless but it might be necessary to keep

Reply:
GitHub should only display changes to the head diff, but anyway fine.